jib-core/src/main/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilder.java

```diff
@@ -113,7 +113,7 @@ public UnwrittenLayer build() {
     TarStreamBuilder tarStreamBuilder = new TarStreamBuilder();
     for (TarArchiveEntry entry : sortedFilesystemEntries) {
       // Strips out all non-reproducible elements from tar archive entries.
-      entry.setModTime(0);
+      entry.setModTime(1);
```

Comment:
I think we decided to use 1000 instead of 1.

Reply:
I fully agree. I just tried to sign the contributors license agreement. Please feel free to close this PR.

Reply:
Okay, will do that. We still appreciate the contribution!